app/screens/home/RouteList.tsx

```diff
@@ -861,6 +870,57 @@ const PaymentDetails = (): ReactElement => {
   );
 };
 
+const StakingInformationTable = ({ balance, statistic }: { balance?: number; statistic?: Statistic }): ReactElement => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <SpacerV />
+      <DataTable>
+        <CompactHeader>
+          <CompactTitle>{t("model.route.statistics")}</CompactTitle>
+        </CompactHeader>
+        <CompactRow>
+          <CompactCell>{t("model.route.balance")}</CompactCell>
+          <CompactCell>{`${formatAmount(balance)} €`}</CompactCell>
+        </CompactRow>
+        <CompactRow>
+          <CompactCell>APR</CompactCell>
+          <CompactCell>{`${(statistic?.staking.yield.apr ?? 0) * 100}%`}</CompactCell>
+        </CompactRow>
+        <CompactRow>
+          <CompactCell>APY</CompactCell>
+          <CompactCell>{`${(statistic?.staking.yield.apy ?? 0) * 100}%`}</CompactCell>
+        </CompactRow>
+      </DataTable>
+      <SpacerV />
+    </>
+  );
+};
+
+const VolumeInformationTable = ({ info }: { info?: VolumeInformation }): ReactElement => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <SpacerV />
+      <DataTable>
+        <CompactHeader>
+          <CompactTitle>{t("model.route.statistics")}</CompactTitle>
+        </CompactHeader>
+        <CompactRow>
+          <CompactCell>{t("model.route.total_volume")}</CompactCell>
+          <CompactCell>{`${formatAmount(info?.total)} €`}</CompactCell>
+        </CompactRow>
+        <CompactRow>
+          <CompactCell>{t("model.route.annual_volume")}</CompactCell>
+          <CompactCell>{`${formatAmount(info?.annual)} €`}</CompactCell>
+        </CompactRow>
+      </DataTable>
+      <SpacerV />
+    </>
+  );
+};
+
```

Comment:
How about joining these two and getting the respective data from a method (similar to cryptoRouteData)?

Reply:
yeah good idea, would then be easier to extend if needed